mcp/mcp.c

```diff
@@ -190,65 +181,25 @@ void * lj_connection_handler(void *arg) {
     // LABJACKS
     blast_info("I am now in charge, initializing LJs");
     // Set the queue to allow new set
+    // leaving the queue in here because it will be used in the future.
     CommandData.Labjack_Queue.set_q = 1;
     CommandData.Labjack_Queue.lj_q_on = 0;
     for (int h = 0; h < NUM_LABJACKS; h++) {
         CommandData.Labjack_Queue.which_q[h] = 0;
     }
-    // init labjacks, first 2 args correspond to the cryo LJs, the next 3 are OF LJs
-    // last argument turns commanding on/off
-    // arguments are 1/0 0 off 1 on
-    // order is CRYO1 CRYO2 OF1 OF2 OF3
-    init_labjacks(1, 1, 1, 1, 1, 1);
-    mult_labjack_networking_init(LABJACK_MULT_OF, 84, 1);
-    mult_labjack_networking_init(LABJACK_MULT_PSS, 84, 1);
-    // 7 is for highbay labjack
-    labjack_networking_init(7, 14, 1);
-    initialize_labjack_commands(7);
-    // initializes an array of voltages for load curves
-    init_array();
-    labjack_networking_init(9, 14, 1);
-    initialize_labjack_commands(9);
-    // switch to this thread for flight
-    mult_initialize_labjack_commands(5);
-    // labjack_networking_init(10, 14, 1);
-    // initialize_labjack_commands(10);
-    ph_thread_t *cmd_thread = mult_initialize_labjack_commands(6);
-    ph_thread_join(cmd_thread, NULL);
-
+    // commented out but left in as a model for how to end this thread
+    // to prevent seg faults.
+    // ph_thread_t *cmd_thread = mult_initialize_labjack_commands(6);
+    // ph_thread_join(cmd_thread, NULL);
     return NULL;
 }
 
 unsigned int superframe_counter[RATE_END] = {0};
 
-static void mcp_488hz_routines(void)
-{
-#ifndef NO_KIDS_TEST
-    write_roach_channels_488hz();
-#endif
-    add_roach_tlm_488hz();
-
-    share_data(RATE_488HZ);
-    framing_publish_488hz();
-    add_frame_to_superframe(channel_data[RATE_488HZ], RATE_488HZ, master_superframe_buffer,
-                            &superframe_counter[RATE_488HZ]);
-}
 
-static void mcp_244hz_routines(void)
-{
-//    write_roach_channels_244hz();
-
-    share_data(RATE_244HZ);
-    framing_publish_244hz();
-    add_frame_to_superframe(channel_data[RATE_244HZ], RATE_244HZ, master_superframe_buffer,
-                            &superframe_counter[RATE_244HZ]);
-}
 
 static void mcp_200hz_routines(void)
 {
-    hawkeye_control(1);
-    outer_frame_200hz(1);
-    process_sun_sensors();
```

Comment:
@ianlowe13 what happened to sun sensors?

Reply:
same as above